spec/genre_spec.rb

```diff
@@ -0,0 +1,60 @@
+require './classes/genre'
+require './classes/item'
+require 'date'
+
+describe Genre do
+  context 'When instantiated' do
+    before(:all) do
+      @genre = Genre.new('old skool')
+      today = Date.today
+      twelve_years = Date.new(today.year - 12, today.month, today.day)
+      @item = Item.new(twelve_years)
+      @item2 = Item.new(today)
+    end
+
+    it 'return the same length if item is dublicated' do
+      @genre.add_item(@item)
+      expect(@genre.items.length).to be(1)
+    end
+
+    it 'return the length of items' do
+      @genre.add_item(@item2)
+      expect(@genre.items.length).to be(2)
+    end
+
+    it 'instance of name' do
+      instance_property = @genre.name
+      expect(instance_property).to eq('old skool')
+    end
+
+    it 'instance of id' do
+      instance_property = @genre.id
+      expect(instance_property).not_to eq(-100)
+    end
+  end
+end
+
+describe Genre do
+  context 'When instantiated' do
+    before(:all) do
+      @genre = Genre.new('old skool')
+      today = Date.today
+      twelve_years = Date.new(today.year - 12, today.month, today.day)
```

Comment:
If you want today's Date, you can use:

```
today = DateTime.now
```

If you want today's date minus 12 years, you can do:

```
twelve_years = DateTime.now.next_year(-2)
```

This way we all keep using DateTime and the app doesn't break.

Reply:
Thank you very much friend  for your constructive reviews, All suggested changes have been implemented.